netty/src/test/java/io/grpc/netty/NettyAdaptiveCumulatorTest.java

```diff
@@ -546,6 +546,7 @@ public CompositeByteBuf addFlattenedComponents(boolean increaseWriterIndex,
         assertEquals(0, compositeThrows.numComponents());
       } finally {
         compositeThrows.release();
+        in.release();
```

Comment:
I think we should keep it above`fail("Cumulator didn't throw");`

Reply:
It is expected the in the case of an error that the ownership remained with the calling method so it is this method's responsibility to do the release.